assets/keymaps/default-windows.json

```diff
@@ -0,0 +1,681 @@
+[
+  // Standard Windows bindings
+  {
+    "bindings": {
```

Comment:
Is this keymap particularly different from the linux keymap? If we can use the linux keymap in both places, that would be preferable :)

Reply:
| key | windows | linux 
| --- | --- | --- 
| shift-alt-up | editor::DuplicateLineUp | editor::AddSelectionAbove
| shift-alt-down | editor::DuplicateLineDown | editor::AddSelectionBelow
| shift-alt-f | editor::Format | null
| ctrl-alt-shift-up | editor::AddSelectionAbove | editor::DuplicateLineUp
| ctrl-alt-shift-down | editor::AddSelectionBelow | editor::DuplicateLineDown
| ctrl-f12 | editor::GoToImplementation | editor::GoToTypeDefinition
| shift-f12 | editor::GoToTypeDefinition | editor::GoToImplementation
| ctrl-shift-f12 | editor::GoToImplementationSplit | editor::GoToDefinitionSplit (It's  ctrl-shift-f10 in default-linux.json (typo?))

Reply:
I also set `ctrl-k` to null to override the default value `editor::CutToEndOfLine`, which ensures that `ctrl-k` key combinations won't accidentally cut.